### DISCUSSION TITLE
how to add School emblem in the /page/about.md

### DISCUSSION THREAD

**Discussion by Connor-Shen** [alshedivat/al-folio#1940] [Q&A] [answered]
To be specific, I add my education and career in my _pages/about.md file. Now I want to add the School emblem and company icon beside the name. How should I do this?
Here is my code in the markdown file, I want to add a small image beside "#### X University" to make it more visible.
#### X University
* Senior Student
#### X Company
* Algorithm Intern

Thank you for your time and help! All advice will be greatly appreciated!

**Comment by george-gca** [comment, marked as answer]
Since it is in the `about` page, I would suggest you to first try adding it using only [markdown](https://github.com/alshedivat/al-folio/issues/1055#issuecomment-1524317980) (assuming you want this before the university name). If you would want it in a different position, say after the name of the university in the rightmost part, you should use html for this. In this case, I would suggest you use the `figure.html` solution with custom css classes. You can check how to use this in the [related blog post](https://alshedivat.github.io/al-folio/blog/2015/images/).

**Reply by Connor-Shen** [reply]
Thanks a lot! Actually, I just add the `<img src="{{ '/assets/img/school.png' | relative_url }}" width="100" style="vertical-align: middle;">` html code before the school part and it works. Thanks again for your time!
